app/pb/communities.proto

```diff
@@ -103,6 +107,17 @@ message ListCommunitiesRes {
   string next_page_token = 2;
 }
 
+message ListUserCommunitiesReq {
+  uint32 page_size = 2;
+  string page_token = 3;
```

Comment:
not sure if I need to keep the page size info in here since I'm not sure how this data will be displayed/fetched for the front end

Reply:
Yes, the frontend will do pagination, either through explicit pages or infinite scroll automagically.